catalog/README.md

```diff
@@ -35,3 +35,73 @@ but that's handled in a separate domain (`state.route`) via reducer composition;
 ### Fetch
 - An accurate check for a successful fetch() would include checking that the promise resolved, then checking that the Response.ok property has a value of true. The code would look something like this:
 [msdn fetch doc](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)
+
+## Deployment (for Quilt internal usage)
+```sh
+firebase use ENV
+firebase deploy --only hosting
+```
+
+### firebase.json
```

Comment:
I'd prefer to have a file `internals/firebase.default.json` or `internals/firebase.placeholder.json`

Reply:
we can just put it into catalog root if that's where it's used from

Reply:
I put it in internals/firebase.default.json for now.

Reply:
It is used from /catalog, btw. Lmk.